css/styles.css

```diff
@@ -0,0 +1,680 @@
+    font-family: 'Lato', sans-serif;
+    font-weight: 400;
+    color: var(--black-color);
+  }
+
+  .nav-link:hover {
+    color: #ec5242;
+    background-color: transparent;
+    border: 3px solid #ec5242;
+    overflow: hidden;
+  }
+
+  .emailpara {
+    width: 50%;
+    margin-left: 40%;
+    margin-top: 1%;
+    text-decoration-line: underline;
+    padding-bottom: 1%;
+    font-family: 'Lato', sans-serif;
+    font-weight: 700;
+    color: var(--black-color);
+  }
+
+  .logosection {
+    margin: auto;
+    width: 50%;
+    align-items: center;
+    text-align: center;
+  }
+
+  .prevhead {
+    margin: auto;
+    width: 50%;
+    text-align: center;
+    font-size: 1.5em;
+    font-family: 'Lato', sans-serif;
+    font-weight: 700;
+    color: var(--black-color);
+  }
+
+  .prevp {
+    margin: auto;
+    margin-top: 3em;
+    width: 50%;
+    text-align: center;
+    color: var(--black-color);
+    font-size: 1em;
+    font-family: 'Lato', sans-serif;
+    font-weight: 700;
+  }
+
+  .farmimgpartners {
+    background-color: #242525;
+  }
+
+  .sumitpara {
+    font-family: 'lato', sans-serif;
+    font-weight: 700;
+    color: #d3d3d3;
+    text-align: center;
+    font-size: large;
+  }
+
+  .sumithead {
+    text-align: center;
+    padding-top: 1em;
+    font-weight: 700;
+    color: #d3d3d3;
+    font-size: xx-large;
+    margin-top: 21px;
+  }
+
+  .speaker1,
+  .speaker2,
+  .speaker3,
+  .speaker4,
+  .speaker5,
+  .speaker6 {
+    display: flex;
+  }
+
+  .logosection h3 {
+    margin-top: 5%;
+    font-size: 1.5em;
+    font-family: 'Lato', sans-serif;
+    font-weight: 700;
+    color: var(--black-color);
+  }
+
+  .logosection hr,
+  .prevsumits hr {
+    width: 10%;
+  }
+
+  .logoimage1 img {
+    width: 36%;
+    margin-bottom: 4em;
+    margin-top: 4em;
+  }
+
+  .sumit2013-img {
+    background-color: rgba(236, 82, 66, 0.75);
+    margin: -21px 0 0 0;
+    height: 100%;
+  }
+
+  .sumit2013 {
+    width: 50%;
+    background-image: url(../img/Next-Generation-Conference-2022-group-photo-c-NFU.jpg);
+    background-repeat: no-repeat;
+    background-size: cover;
+    backdrop-filter: blur(10px);
+    height: 13em;
+  }
+
+  .sumit2014 {
+    width: 50%;
+    background-image: url(../img/PXL_20211118_092531580-scaled-e1638366981737.jpg);
+    background-repeat: no-repeat;
+    background-size: cover;
+    height: 13em;
+  }
+
+  .left-footer-text {
+    font-size: small;
+    font-family: 'Lato', sans-serif;
+    text-align: left;
+    margin: -10px 0 0 78px;
+    width: 70%;
+    color: #272a31;
+    margin-left: 32%;
+  }
+
+  .left-footer-textabout {
+    font-size: small;
+    font-family: 'Lato-Bold', sans-serif;
+    text-align: left;
+    margin: -10px 0 0 78px;
+    width: 70%;
+    color: #d3d3d3;
+    margin-left: 32%;
+  }
+
+  .footer-section {
+    height: 11em;
+  }
+
+  .footer-section-container {
+    display: flex;
+    width: 70%;
+    padding-top: 3em;
+    margin: auto;
+    margin-bottom: 5%;
+  }
```

Comment:
- [ ] KIndly make sure your `footer` on the home page is white and on `about page is dark( make sure to the same dark color used on the provided design)

Reply:
thank you for noticing that part, fixed it. 
the about page is using D3d3d3
the home page footer is using 272a31